pkgs/tools/misc/fluent-bit/default.nix

```diff
@@ -0,0 +1,23 @@
+{ pkgs, stdenv, fetchgit, cmake, ... }:
```

Comment:
Please do not include `pkgs` as an argument in a derivation. There is also no need for `...` in this case.